linear_search/linear_search.js

```diff
@@ -0,0 +1,29 @@
+function linearSearch (arr, x) {
+	/*
+	Performs a linear search
+	:param arr: List of elements to search from
+	:param x: Element to search for
+	:return: Index if element found else -1
+	*/
+	for (let it = 0; it < arr.length; it++) {
+		if (arr[it] === x) {
+			return it;
+		}
+	}
+	return -1;
+}
+
+function main () {
+	let A = [];
```

Comment:
Don't use ALLCAPS for local variables and use a general word instead of `A`, i.e. `let data =[];`